### ISSUE TITLE
The notes move after the cutscene

### ISSUE THREAD

**Issue by Gaster123987** [ShadowMario/FNF-PsychEngine#15900]
### Describe your problem here.

Welp, just gonna make this short, as the title suggest for some reason the notes move after the intro cutscene for an song.
Here is what happens after the cutscene
![Zrzut ekranu 2024-11-12 162308](https://github.com/user-attachments/assets/1414cc27-9177-4482-bcb4-fdaec49bd1d5)
Also, I don't know if it is related to the issue or not, but when I restart the song this error pops up
![Zrzut ekranu 2024-11-12 180713](https://github.com/user-attachments/assets/ad87999a-f000-4036-b4bd-886e8eee792d)

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows x64

### Did you edit anything in this build? If so, mention or summarize your changes.

I olny removed the weeks that were present.

**Comment by Shadow06735** [comment]
do you have any script that modify's the note positions?

**Comment by Gaster123987** [comment]
I don't think so? Unless you count the script that makes the opponents notes just not visible

**Comment by Shadow06735** [comment]
> I don't think so? Unless you count the script that makes the opponents notes just not visible

I noticed how you use an old tv static for visual effect. Maybe that could be causing the notes to move to the side?

**Comment by Gaster123987** [comment]
I just removed those and tried again, and nope, it's still bugged even when the shader is removed

**Comment by Gaster123987** [comment]
![Zrzut ekranu 2024-11-13 181949](https://github.com/user-attachments/assets/8fb8a4ce-9546-4eb4-9cb7-63ca670be0ca)
Also the notes splashes are for some reason in the wrong spot and I have no idea how to fix that either

**Comment by Shadow06735** [comment]
> ![Zrzut ekranu 2024-11-13 181949](https://private-user-images.githubusercontent.com/136098961/386266857-8fb8a4ce-9546-4eb4-9cb7-63ca670be0ca.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE2MDQ3MjcsIm5iZiI6MTczMTYwNDQyNywicGF0aCI6Ii8xMzYwOTg5NjEvMzg2MjY2ODU3LThmYjhhNGNlLTk1NDYtNGViNC05Y2I3LTYzY2E2NzBiZTBjYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExNFQxNzEzNDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02YmUxY2Q1ODQ3ZTUwYjRjMTQxMjZlNTU2ZDJkOTgzMTY3YzEyYzQ2YzhjNTYxZDFiY2E2NGFmNzRjZjc0NzE1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.NFWflPek7CP3-jAe0VFawPu5EMSsJrHNugHGQVIyPEw) Also the notes splashes are for some reason in the wrong spot and I have no idea how to fix that either

when you mean the notes move after cutscene, u mean that without the cutscene it should be fine?

**Comment by Gaster123987** [comment]
Yeah, specificly without the script that I use that causes the cutscene
Here is the code just in case
`local allowCountdown = false
function onStartCountdown()
        if not allowCountdown and isStoryMode and not seenCutscene then --Block the first countdown
                startVideo('HopelessIntro');
                allowCountdown = true;
                return Function_Stop;
        end
        return Function_Continue;
end`

**Comment by Gaster123987** [comment]
Also, I'm gonna make an separate ticket for the Crashing after restart thing
